test/cat.jl

```diff
@@ -1,18 +1,17 @@
 module TestCat
-    using Base.Test
-    using DataTables
+    using Base.Test, DataTables, Nulls
 
     #
     # hcat
     #
 
-    nvint = NullableArray(Nullable{Int}[1, 2, Nullable(), 4])
-    nvstr = NullableArray(Nullable{String}["one", "two", Nullable(), "four"])
+    nvint = Vector(?(Int)[1, 2, null, 4])
```

Comment:
You don't need to specify the types here and in all other tests below. These were required on Julia 0.5, but it's a good occasion to make the tests more readable.